.github/workflows/ci.yml

```diff
@@ -19,5 +19,7 @@ jobs:
         run: yarn test:ci
       - name: Typescript build
         run: yarn build
-      - name: Storybook build
-        run: yarn build-storybook
+      - name: Run Chromatic
+        run: yarn chromatic
+        env:
+          CHROMATIC_PROJECT_TOKEN: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```

Comment:
The logs from the run indicate its better to use their published action instead of running this manually

Reply:
https://github.com/marketplace/actions/publish-to-chromatic